admin/client/App/screens/Item/components/RelatedItemsList.js

```diff
@@ -47,48 +50,69 @@ const RelatedItemsList = React.createClass({
 	},
 	renderItems () {
 		return this.state.items.results.length ? (
-			<div className="ItemList-wrapper">
-				<table cellPadding="0" cellSpacing="0" className="Table ItemList">
-					{this.renderTableCols()}
-					{this.renderTableHeaders()}
-					<tbody>
-						{this.state.items.results.map(this.renderTableRow)}
-					</tbody>
-				</table>
+			<div>
+				{this.renderTableCols()}
+				{this.state.items.results.map(this.renderTableRow)}
 			</div>
 		) : (
 			<h4 className="Relationship__noresults">No related {this.props.refList.plural}</h4>
 		);
 	},
-	renderTableCols () {
-		const cols = this.state.columns.map((col) => <col width={col.width} key={col.path} />);
-		return <colgroup>{cols}</colgroup>;
+	renderRelationshipColumn (item) {
+		return <td key={'Relationship' + item.id || ''}>{this.props.relationship.label || titlecase(this.props.relationship.path)}</td>;
 	},
-	renderTableHeaders () {
-		const cells = this.state.columns.map((col) => {
-			return <th key={col.path}>{col.label}</th>;
-		});
-		return <thead><tr>{cells}</tr></thead>;
+	renderParentColumn (item) {
+		const listHref = `${Keystone.adminPath}/${this.props.refList.path}`;
+		return <td key={'Parent' + item.id} className="Relationship__link"><a href={listHref}>{this.props.refList.label}</a></td>;
+	},
+	renderNameColumn (item) {
+		const column = this.state.columns[0];
+		const ColumnType = Columns[column.type] || Columns.__unrecognised__;
+		const linkTo = `${Keystone.adminPath}/${this.props.refList.path}/${item.id}`;
+		return <ColumnType key={column.path} list={this.props.refList} col={column} data={item} linkTo={linkTo} />;
+	},
+	renderFieldColumn (item) {
+		const listHref = `${Keystone.adminPath}/${this.props.refList.path}`;
+		const linkValue = this.state.columns[1] ? <a href={listHref}>{this.state.columns[1].label}</a> : null;
+		return <td key={'Field' + item.id} className="Relationship__link">{linkValue}</td>;
+	},
+	renderValueColumn (item) {
+		const column = this.state.columns[1];
+		const ColumnType = Columns[column.type] || Columns.__unrecognised__;
+		const linkTo = `${Keystone.adminPath}/${this.props.refList.path}/${item.id}`;
+		return <ColumnType key={column.path} list={this.props.refList} col={column} data={item} linkTo={linkTo} />;
 	},
 	renderTableRow (item) {
-		const cells = this.state.columns.map((col, i) => {
-			const ColumnType = Columns[col.type] || Columns.__unrecognised__;
-			const linkTo = !i ? `${Keystone.adminPath}/${this.props.refList.path}/${item.id}` : undefined;
-			return <ColumnType key={col.path} list={this.props.refList} col={col} data={item} linkTo={linkTo} />;
-		});
-		return <tr key={'i' + item.id}>{cells}</tr>;
+		return (
+			<tr key={'i' + item.id}>{[
+				this.renderRelationshipColumn(item),
+				this.renderParentColumn(item),
+				this.renderNameColumn(item),
+				this.renderFieldColumn(item),
+				this.renderValueColumn(item),
+			]}</tr>
```

Comment:
A key of `i` could be non-unique, thus lead to confusion and unnecessary rerenders from React. Maybe use `'table-row-item-' + item.id` or something like that?